### PR TITLE
big cleaning of invitation & events

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/event/src/lib/+state/event.service.ts [hunk 1]
```diff
@@ -97,8 +95,9 @@ export class EventService extends CollectionService<EventState> {
 
     // remove possible undefined values to avoid
     // FirebaseError: [code=invalid-argument]: Unsupported field value: undefined
-    if (!e.movie) delete e.movie;
-    if (!e.organizedBy) delete e.organizedBy;
+    if (!!e.movie) delete e.movie;
+    if (!!e.organizedBy) delete e.organizedBy;
+    if (!!e.org) delete e.org;
```
Comment (Mussuranae) [inline libs/event/src/lib/+state/event.service.ts:100]: Is it not `if (!e.movie)` with only one `!` ? We want to delete the field is there is no value right ?

File: libs/firebase-utils/src/lib/firestoreMigrations/0053.ts [hunk 2]
```diff
@@ -12,6 +12,21 @@ export async function upgrade(db: Firestore) {
...
+
+    if (!!data.meta?.organizerId) {
+      data.meta.organizerUid = data.meta.organizerId;
+      delete data.meta.organizerId;
+    }
+
+    // Cleaning fields that should not be here CF formatToFirestore in event.service
+    if (!!data.movie) delete data.movie;
+    if (!!data.organizedBy) delete data.organizedBy;
+    if (!!data.org) delete data.org;
```
Comment (Mussuranae) [inline libs/firebase-utils/src/lib/firestoreMigrations/0053.ts:28]: Sooo, if there is a movie on the event, we delete it ?

**Review by Mussuranae — APPROVED** [review]

File: libs/firebase-utils/src/lib/firestoreMigrations/0053.ts (hunk 2 above)
Comment (phpgeekfr) [reply libs/firebase-utils/src/lib/firestoreMigrations/0053.ts:28]: yes, this is here to clean the bug introduced (and corrected in this PR ) in `libs/event/src/lib/+state/event.service.ts `

We had objects on invitations that are supposed to be only in fron

File: libs/event/src/lib/+state/event.service.ts (hunk 1 above)
Comment (phpgeekfr) [reply libs/event/src/lib/+state/event.service.ts:100]: I think this was a typo. invitation.movie or invitation.organizeBy & invitation.org are only for front & should be deteled when we save on db

File: libs/firebase-utils/src/lib/firestoreMigrations/0053.ts (hunk 2 above)
Comment (Mussuranae) [reply libs/firebase-utils/src/lib/firestoreMigrations/0053.ts:28]: Ah ok, thanks !

File: libs/event/src/lib/+state/event.service.ts (hunk 1 above)
Comment (Mussuranae) [reply libs/event/src/lib/+state/event.service.ts:100]: perfect then !

**Pull request merged**